examples/storybook/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "storybook",
+  "version": "0.0.1",
+  "scripts": {
+    "start": "start-storybook -p 9001 -c .storybook",
+    "build": "build-storybook -c .storybook -o dist/"
+  },
+  "private": true,
+  "dependencies": {
+    "@angular/common": "7.0.0",
+    "@angular/compiler": "7.0.0",
+    "@angular/core": "7.0.0",
+    "@angular/forms": "7.0.0",
+    "@angular/http": "7.0.0",
+    "@angular/platform-browser": "7.0.0",
+    "@angular/platform-browser-dynamic": "7.0.0",
+    "@angular/router": "7.0.0",
+    "angular-instantsearch": "2.1.0",
+    "core-js": "2.5.4",
+    "rxjs": "6.3.3",
+    "zone.js": "0.8.26"
+  },
```

Comment:
I would remove some of theses dependencies but I'm not sure yet storybook isn't using them.
I suspect some of them are silently used by storybook (I broke it by removing @angular/forms)
Once I've finished loading all stories I'll remove them

Reply:
done in #269